documentation/2.0/content/userguide/tools-config/domain_def.md

```diff
@@ -185,3 +185,134 @@ weight: 2
  ```
 
  The `targeting` attribute is not valid for WebLogic Server versions 12.2.1 and up.
+
+#### Using the typedef file to exclude template installed resources from the model and archive
+
+ The Discover tool attempts to provide a sparse model by doing such things as not including attributes that are default values. It is important to not install template resources through the model. First, the templates will install the same resources in the target domain . Second, if you install resources from a on-premise domain into a target domain that is a later WebLogic version, it could cause conflicts.
+ The `system-elements` section of the typedef file is used to assist the Discover tool. The `system-elements` section is a list that tells discover what to exclude from the model and archive files.
```

Comment:
Discover tool -> Discover Domain Tool
section is a list -> comprises a list
that tells discover -> that tells the tool